src/include/executor/executor_context.h

```diff
@@ -44,6 +45,8 @@ class ExecutorContext {
 
   type::EphemeralPool *GetPool();
 
+  std::string GetDatabaseName() const;
```

Comment:
Like previous comment, you can make this function inline to return the database name (like what other files do)

Reply:
This is what other functions in the same file do, so gonna keep it like this.